LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Kenny Levinsen
+Copyright (c) 2016-2017 Kenny Levinsen
```

Comment:
Could you let it be just "2017" if you want to update it? Not that it legally matters, copyright is entirely implicit, and this text has no effect at all ("all rights reserved" was required many years ago in some obscure countries, but no more). Technically, it's not even correct anymore, as I don't hold the rights to the code you provide without a longer formal legal dance. :)

This is also a really good time to come with a small nugget of wisdom: Focus on the primary task when making a change. It's a little bit silly to have a change blocked or delayed by such a small thing. Keeping PR's (github/bitbucket/...), changesets (gerrit) focused provides huge benefits, both in development efficiency, review efficiency (reviewers don't have to wonder about changes), debugging/testing efficiency (things that break are directly related to the change), version control history and just overall reduce frustrations.